zhaquirks/inovelli/__init__.py

```diff
@@ -66,97 +69,42 @@
 # 1 - down button
 # 2 - up button
 # 3 - config button
+# 4 - aux down button
+# 5 - aux up button
+# 6 - aux config button
 
-BUTTONS = {1: BUTTON_1, 2: BUTTON_2, 3: BUTTON_3}
+BUTTONS = {1: BUTTON_1, 2: BUTTON_2, 3: BUTTON_3, 4: BUTTON_4, 5: BUTTON_5, 6: BUTTON_6}
 ON = "Up"
 OFF = "Down"
 CONFIG = "Config"
+AUX_ON = "Aux Up"
+AUX_OFF = "Aux Down"
+AUX_CONFIG = "Aux Config"
 
 NOTIFICATION_TYPE = "notification_type"
 
 
-class Inovelli_VZM31SN_Cluster(CustomCluster):
-    """Inovelli VZM31-SN custom cluster."""
-
-    cluster_id = 0xFC31
-    name = "InovelliVZM31SNCluster"
-    ep_attribute = "inovelli_vzm31sn_cluster"
+class Inovelli_Cluster(CustomCluster):
```

Comment:
It's preferred to use "CamelCase" / "CapWords" for class names.

Reply:
From what I can see, `cluster_id` and `ep_attribute` is the same for all Inovelli cluster classes.
Can we just put it in this base class then?

Reply:
> saying the preference would be to do "InovelliCluster" as opposed to "Inovelli_Cluster"? 

Yes. That's [preferred](https://peps.python.org/pep-0008/#class-names). 

> Putting cluster_id and ep_attribute in the base class is giving async_initialize errors...it all appears to work, but putting them across each cluster resolved that and is why I ended up splitting them out.

Weird. There shouldn't be any difference. Are you sure it wasn't just some connectivity issue?

Reply:
The log level was changed indeed. Are there more (debug) log messages when `zigpy` is also set to debug level?
There's no reason I can see why this could happen. Setting `cluster_id`, `name`, and `ep_attribute` in the "base class" vs the one inheriting that doesn't make a difference.
Can you double-confirm that the error happens every time? And not every time when set in the base classes?

Also, did you check that all attributes set here are correct?
https://github.com/home-assistant/core/blob/567a179084e0226a45d1db01ffb5d4e2e37f3d8c/homeassistant/components/zha/core/cluster_handlers/manufacturerspecific.py#L249-L253
(There might be more info in zigpy debug logs)

Reply:
Possibly related comment: https://github.com/zigpy/zha-device-handlers/pull/2960/files#r1485413406